test/e2e/cluster_upgrade_runtimesdk_test.go

```diff
@@ -50,7 +50,9 @@ var _ = Describe("When upgrading a workload cluster using ClusterClass with Runt
 				framework.ValidateResourceVersionStable(ctx, proxy, namespace, clusterctlcluster.FilterClusterObjectsWithNameFilter(clusterName))
 			},
 			// "upgrades" is the same as the "topology" flavor but with an additional MachinePool.
-			Flavor: ptr.To("upgrades-runtimesdk"),
+			Flavor:               ptr.To("upgrades-runtimesdk"),
+			ExtensionNamespace:   "test-extension-system",
```

Comment:
Let's add a short comment what this is referring to (basically where the source of truth is that the RX is actualy depoyed in that namespace & with that service name)